src/healthcare-application/schema.js

```diff
@@ -207,6 +207,9 @@ module.exports = {
     stateOfBirth: {
       'enum': states
     },
+    placeOfBirthState: {
+      'enum': states + [{ label: 'Other', value: 'Other' }]
```

Comment:
Ah, I see re: https://github.com/department-of-veterans-affairs/vets-api/pull/696/commits/d0553cc88087173d1c9fec5e8fbda000c80f827c#r95842564

What I'd meant in our chat was that `states + [{ label: 'Other', value: 'Other' }]` could be the enum for `stateOfBirth`. I just wanted to avoid redefining `states` itself.